packages/in_app_purchase/example/ios/in_app_purchase_pluginTests/PaymentQueueTest.m

```diff
@@ -1,183 +0,0 @@
-// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
@cyanglaz This test doesn't compile and wasn't included in the Xcode project, so I deleted it.  I can file an issue for it to be re-added and fixed to pass, if you still value in it.

Reply:
Yes, please add this back and remove the .testing part. I am not sure why it would be like this in the first place.

Reply:
Done.